src/utils.py

```diff
@@ -55,12 +55,17 @@ def parse_fpg(filename):
             # as delimiters where tabs should have been used. Rectify that
             # here.
             line = line.replace(' ', '\t')
-            LOGGER.warn('An old version of mlib_devel generated %s. Please'
+            LOGGER.warn('An old version of mlib_devel generated %s. Please '
                         'update.' % filename)
             # and carry on as usual.
-            line = line.replace('\_', ' ').replace('?meta ', '')
+            line = line.replace('\_', ' ').replace('?meta', '')
             line = line.replace('\n', '').lstrip().rstrip()
-            name, tag, param, value = line.split('\t')
+            lineSplit = line.split('\t')
```

Comment:
http://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variable-and-function-names

Our convention so far has been camelCase in class names, but underscores everywhere else.